### PR TITLE
[RFR] Add Command logger

### PR DESCRIPTION
**Pull request by jeromemacias**
Fix https://github.com/Hexanet/MonologExtraBundle/issues/30

### CODE REVIEW

File: src/Logger/Command/CommandLogger.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CommandLogger implements CommandLoggerInterface
+{
+    /**
+     * LoggerInterface
```
Comment (notFloran) [inline src/Logger/Command/CommandLogger.php]: Can you fix the missing `@var` ? I will fix the issues in the other classes

File: src/Logger/Command/CommandLogger.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+     * Construct
+     *
+     * @param LoggerInterface $logger Logger
+     */
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
+    public function logCommand(Command $command, InputInterface $input, OutputInterface $output) : void
```
Comment (notFloran) [inline src/Logger/Command/CommandLogger.php]: Can you add a phpdoc ?

File: src/Logger/Command/CommandLoggerInterface.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+namespace Hexanet\Common\MonologExtraBundle\Logger\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+interface CommandLoggerInterface
+{
+    public function logCommand(Command $command, InputInterface $input, OutputInterface $output) : void;
```
Comment (notFloran) [inline src/Logger/Command/CommandLoggerInterface.php]: Can you add a phpdoc ?

**Review by notFloran — CHANGES_REQUESTED** [review]
Thanks a lot for this contribution.

I made some comments related to the phpdoc, other than that's good.

**Comment by jeromemacias** [comment]
@notFloran Review done, even for `@var LoggerInterface` in other classes ;)

**Review by notFloran — APPROVED** [review]

**Review by nclsHart — APPROVED** [review]
LGTM

**Pull request merged**